src/test/codewhisperer/util/authUtil.test.ts

```diff
@@ -128,11 +128,11 @@ describe('AuthUtil', async function () {
         await authUtil.connectToAwsBuilderId()
         assert.ok(authUtil.isConnected())
         assert.ok(authUtil.isConnectionValid())
-        assert.ok(isSsoConnection(authUtil.conn))
+        assert.ok(isBuilderIdConnection(authUtil.conn))
```

Comment:
why wasn't this changed to `isAnySsoConnection` like the other cases above?

Reply:
This test uses a builder id in this case